src/containers/ConversionForm/index.tsx

```diff
@@ -123,8 +164,12 @@ const ConversionForm = ({
                 <Button ghost>Cancel</Button>
                 <Button
                     type="primary"
-                    disabled={!fileToConvert || !engineSelected}
-                    onClick={() => setIsProcessing(!isProcessing)}
+                    disabled={!readyToConvert}
+                    onClick={() =>
+                        readyToConvert
+                            ? validateFileType(fileToConvert.name)
+                            : null
+                    } // this will be unclickable anyway, but typescript doesn't' know that
```

Comment:
```suggestion
                        readyToConvert && validateFileType(fileToConvert.name)
                    } // this will be unclickable anyway, but typescript doesn't know that
```

Nit: Could use the same `&&` condition execution instead of conditional ternary (`?`)